latest/examples/transformations/xarrayLike.json

```
@@ -0,0 +1,17 @@
+{
+    "coordinateSystems": [
+        { "name": "in", "axes": [ {"name": "i"}, {"name": "j"} ]},
+        { "name": "out", "axes": [ {"name": "x"}, {"name": "y"} ]}
```

Could this get annotated with `"type"` fields?

Should this be more like:

```suggestion
        { "name": "in", "axes": [ {"name": "i", "type": "array"}, {"name": "j", "type": "array"} ]},
        { 
            "name": "out", 
            "axes": [ 
                {"name": "x", "type": "coordinate", "discrete": true},
                {"name": "y", "type": "coordinate", "discrete": true} 
            ]
        }
```